### PR TITLE
Add md-progress-bar README and package.json for component.

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: src/components/progress-bar/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<md-progress-bar mode="indeterminate" color="primary"></md-progress-bar>
+<md-progress-bar mode="indeterminate" color="accent"></md-progress-bar>
+<md-progress-bar mode="indeterminate" color="warn"></md-progress-bar>
+ ```
+
+### Accessibility
+
+ * ARIA attributes are applied to the indicator defining the valuemin, valuemax and valuenow attributes.
+
+### Upcoming work
```
Comment (jelbourn) [inline src/components/progress-bar/README.md]: Omit the `Upcoming work` section

File: src/components/progress-bar/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+ * Adding ARIA attribute for progressbar "for".
+
+### API Summary
+
+Properties:
+
+| Name            | Type                                               | Description |
+| ---             | ---                                                | --- |
+| `color`         | `"primary"|"accent"|"warn"`                        | The color palette of the progress indicator |
```
Comment (jelbourn) [inline src/components/progress-bar/README.md]: Add spaces around `|` separators

File: src/components/progress-bar/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/angular/material2.git"
+  },
+  "keywords": [
+    "angular",
+    "material",
+    "material design",
+    "components",
+    "progress"
```
Comment (jelbourn) [inline src/components/progress-bar/package.json]: Add keywords
`progressbar`
`loading`

**Comment by josephperrott** [comment]
Comments addressed, PTAL.

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**